Transfer/Sources/CreateTransferPresenter.swift

```diff
@@ -289,23 +282,9 @@ final class CreateTransferPresenter {
 
     private func updateFooterContent(_ errors: [HyperwalletError]) {
         for error in errors {
-            guard let fieldName = error.fieldName, !fieldName.isEmpty else {
-                continue
-            }
-            switch fieldName {
-            case "amount":
-                if let sectionData = sectionData.first(where: { $0.createTransferSectionHeader == .transfer }) {
-                    sectionData.errorMessage = error.message
-                    view.updateFooter(for: .transfer)
-                }
-            case "notes":
-                if let sectionData = sectionData.first(where: { $0.createTransferSectionHeader == .notes }) {
-                    sectionData.errorMessage = error.message
-                    view.updateFooter(for: .notes)
-                }
-
-            default:
-                break
+            if let sectionData = sectionData.first(where: { $0.createTransferSectionHeader == .transfer }) {
```

Comment:
For now let's show all the errors in the footer of transfer section. Even for notes error will be shown under transfer section. Once Willem comes up with concrete requirement then we will create another ticket to handle all those scenarios.

Reply:
Roger that.